### ISSUE TITLE
Provide better example of succinctness

### ISSUE THREAD

**Issue by daobrien** [StyleGuides/WritingStyleGuide#24]
The example given in this section is not an improvement. It needs rewriting.

⁠ Verbosity
Try not to use unnecessary words. Keep it succinct.
⁠
Table 2.4. 

Example
The individual member of the social community often receives his information via visual, symbolic channels.

Improvement
People read. (Translation by Richard Feynman.)

**Comment by julian-cable** [comment]
Suggestion to replace existing example: 

Example
This example performs a file system check on the XFS file system residing on `/dev/vdb1`.

Improvement
This example checks the XFS file system on `/dev/vdb1`.

**Comment by daobrien** [comment]
Much better than the previous example :-|

Might want to reconsider the use of "check" given that we routinely recommend that writers avoid it (even though the command is `fsck`). Can you rephrase to avoid the object at the end of the sentence? And to help drive the point home, maybe two examples, using two contexts.

**Comment by julian-cable** [comment]
And a further example:

Example
It is possible that the scheduled snapshot takes some time to be created.

Improvement
Creating the scheduled snapshot might take time.

**Comment by julian-cable** [comment]
Added these two new example pairs to Table 2.5, to replace the original example pair.
